doc/user/content/sql/select.md

```diff
@@ -78,7 +75,7 @@ Performing a `SELECT` query that does not directly read out of a materialization
 
 For technical reasons, you are not able to perform a `SELECT` query that depends, even transitively, on non-materialized sources. This is because Materialize must choose a "moment" at which to execute your query, and Materialize does not know which moments are valid for non-materialized sources. You can use the [`SHOW FULL VIEWS`](../show-views) command to report which views may be queried.
```

Comment:
I'm having a hard time squaring how non-materialized sources can be of any use with this statement:

> For technical reasons, you are not able to perform a `SELECT` query that depends, even transitively, on non-materialized sources.

This, to me, sounds like if you use a non-materialized view, it can never be `SELECT`ed at all. While you're looking at this, could I get you to take a swag at improving this line, as well? Maybe it just needs a clause like, "unless there is a materialized view between the dependency and the non-materialized view?"